test/testdata/rbi/basicobject_instance_eval.rb

```diff
@@ -4,14 +4,15 @@ class A
   sig do
```

Comment:
We can just delete this method and call the method declared in `BasicObject`, since `BasicObject` is an ancestor of `A` (I'm not really sure why this test redefines it).